.github/dependabot.yml

```diff
@@ -26,6 +34,10 @@ updates:
         update-types:
         - "version-update:semver-major"
         - "version-update:semver-minor"
+    groups:
+      all:
+        patterns:
+        - "*"
```

Comment:
we shouldn't do * here, but only for things we want to group (for example k8s) but I remember CRI had to be specific version to not break stuff